doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+            "description": "When useAcrylic is set to true, it sets the transparency of the window for the profile. Accepts floating point values from 0-1 (default 0.5).",
+            "maximum": 1,
+            "minimum": 0,
+            "type": "number"
+          },
+          "background": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the background color of the profile. Overrides background set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\". Default \"#000000\" (black)."
+          },
+          "backgroundImage": {
+            "description": "(Not in SettingsSchema.md)",
+            "type": "string"
+          },
+          "backgroundImageAlignment": {
+            "default": "center",
+            "enum": [
+              "bottom",
+              "bottomLeft",
+              "bottomRight",
+              "center",
+              "left",
+              "right",
+              "top",
+              "topLeft",
+              "topRight"
+            ],
+            "type": "string"
+          },
+          "backgroundImageOpacity": {
+            "description": "(Not in SettingsSchema.md)",
+            "maximum": 1,
+            "minimum": 0,
+            "type": "number"
+          },
+          "backgroundImageStretchMode": {
+            "description": "(Not in SettingsSchema.md) Possible values: \"none\", \"fill\", \"uniform\", \"uniformToFill\" (default)",
+            "enum": [
+              "fill",
+              "none",
+              "uniform",
+              "uniformToFill"
+            ],
+            "type": "string"
+          },
+          "closeOnExit": {
+            "default": true,
+            "description": "When set to true (default), the selected tab closes when exit is typed. When set to false, the tab will remain open when exit is typed.",
+            "type": "boolean"
+          },
+          "colorScheme": {
+            "default": "Campbell",
+            "description": "Name of the terminal color scheme to use. Color schemes are defined under schemes.",
+            "type": "string"
+          },
+          "colorTable": {
+            "description": "Array of colors used in the profile if colorscheme is not set. Colors use hex color format: \"#rrggbb\". Ordering is as follows: [black, red, green, yellow, blue, magenta, cyan, white, bright black, bright red, bright green, bright yellow, bright blue, bright magenta, bright cyan, bright white]",
+            "items": {
+              "$ref": "#/definitions/Color"
+            },
+            "maxItems": 16,
+            "minItems": 16,
+            "type": "array"
+          },
+          "commandline": {
+            "default": "powershell.exe",
+            "description": "Executable used in the profile.",
+            "type": "string"
+          },
+          "connectionType": {
+            "$ref": "#/definitions/ProfileKey",
+            "default": "{d9fcfdfa-a479-412c-83b7-c5640e61cd62}",
```

Comment:
Shouldn't this _not_ have a default, as I'm pretty sure the default is the null guid?

Reply:
ABSOLUTELY NOT